src/Psalm/Internal/Codebase/Analyzer.php

```diff
@@ -885,7 +885,7 @@ public function hasMixedMemberName(string $member_id) : bool
     }
 
     /**
-     * @var array<string, string<bool>> $names
+     * @param array<string, string<bool>> $names
```

Comment:
Cannot create generic object with reserved word in docblock for Psalm\Internal\Codebase\Analyzer::addMixedMemberNames
```
     * @param array<string, string<bool>> $names
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
```